source/MRMesh/MRGcodeProcessor.h

```diff
@@ -47,6 +50,20 @@ class MRMESH_CLASS GcodeProcessor
     // process all commands from one line g-code source and generate corresponding move action
     MRMESH_API MoveAction processLine( const std::string_view& line );
 
+    // settings
+    enum class RotationParameterName
+    {
+        None = -1,
+        A,
+        B,
+        C,
+        Count
+    };
+    MRMESH_API void setRotationParams( RotationParameterName paramName, const Vector3f& rotationAxis );
+    MRMESH_API Vector3f getRotationParams( RotationParameterName paramName );
+    MRMESH_API bool setRotationOrder( std::array<RotationParameterName, 3> rotationAxisOrder );
+    MRMESH_API std::array<RotationParameterName, 3> getRotationOrder();
```

Comment:
mb make single structure with gcode axis name to real axis map, and order, and one getter setter for it

Reply:
it's possible, but i don't see serious advantages

Reply:
I think it will be just more friendly interface